actions/datadog-release-metric/DatadogReleaseMetric.ts

```diff
@@ -0,0 +1,90 @@
+import * as core from "@actions/core";
+import { Context } from "@actions/github/lib/context";
+import {
+  MetricsApi,
+  MetricsApiSubmitMetricsRequest,
+} from "@datadog/datadog-api-client/dist/packages/datadog-api-client-v1";
+
+const METRIC_TYPE = "count";
+const METRIC_NAME = "library.release.count";
+const REF_REGEX = /^refs\/(.+?)\/(.+)$/;
+
+export interface MetricParams {
+  type: string;
+  name: string;
+  value: number;
+  tags: string[];
+}
+
+export default class DatadogReleaseMetric {
+  private readonly type: string = METRIC_TYPE;
+  private readonly name: string = METRIC_NAME;
+
+  constructor(private readonly context: Context, private datadog: MetricsApi) {}
+
+  async run() {
+    const params: MetricParams = {
+      type: this.getType(),
+      name: this.getName(),
+      tags: this.getTags(this.getOrg(), this.getRepo(), this.getVersion()),
+      value: 1,
+    };
+    await this.sendMetric(params);
+  }
+
+  getVersion(): string {
```

Comment:
Should move this to a common utils file. Or drop it based on other comment.

Reply:
I'm going to move this to common utils and use it to enforce this action is only run on tags. Still ripping out version tag being pushed to Datadog